routers/docs.py

```diff
@@ -43,16 +43,16 @@ async def generate_file_docs(
         raise HTTPException(status_code=http.client.INTERNAL_SERVER_ERROR)
 
 
-@router.get("/file-docs/{id}")
+@router.get("/file-docs/{doc_id}")
```

Comment:
i changed this because my IDE warned me that the `id` field already existed intrinsically in the function (i guess because of the @router.get decorator, and that we were overwriting it.